lib/clients/OpenstackClient.py

```diff
@@ -264,10 +268,14 @@ def _delete_volume(self, volume_id):
             self.logger.info('{} SUCCESS: volume-id={}'.format(log_prefix, volume_id))
             return True
         except Exception as error:
-            message = '{} ERROR: volume-id={}\n{}'.format(log_prefix, volume_id, error)
-            self.logger.error(message)
-            raise Exception(message)
-
+            if error.code == 404:
```

Comment:
If possible define 404 in a constant, in BaseClient.py or seperate constant file.